src/reanimated2/threads.ts

```diff
@@ -169,7 +169,9 @@ export function runOnJS<A extends any[], R>(
   }
   return (...args) => {
     _scheduleOnJS(
+      // @ts-ignore TODO TYPESCRIPT omitted in the past
```

Comment:
You mean sth like "TODO TYPESCRIPT One of bugs ignored when introducing types"? 🤔  Could you please rewrite it?

Reply:
This is runOnJS issue I ran into when fixing types that's in #4572.